src/clr-angular/data/datagrid/interfaces/column-state.interface.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { DatagridColumnChanges } from '../enums/column-changes.enum';
+
+export interface DatagridColumnState {
```

Comment:
Is the intent with this to add other things here?  e.g hidden state, master detail mode (if thats the correct description).